public/styles.css

```diff
@@ -1073,6 +1073,7 @@ input[type=number]::-webkit-outer-spin-button {
 .mobile-content .trade-params > div {
     margin-right: .5rem;
     min-width: 10rem;
```

Comment:
I was thinking about the controls variations and forgot that the screen/trade size varies also. I think you have a point considering container width changes with the layout. Controls tend to stretch sometimes when minimum width is set on the parent container. This usually is the case with radio and checkboxes who tend to compensate the minimum width with margins in most cases. I think we are fine in this case now that i check 😉